tests/integration/targets/ec2_group/tasks/group_info.yml

```
@@ -0,0 +1,84 @@
+---
+
+# file for testing the ec2_group_info module
+
+- block:
+    # ======================== Setup =====================================
+    - name: Create a group for testing group info retrieval below
+      ec2_group:
+        name: '{{ ec2_group_name }}-info-1'
+        vpc_id: '{{ vpc_result.vpc.id }}'
+        description: '{{ ec2_group_description }}'
+        rules:
+          - proto: tcp
+            ports:
+              - 90
+            cidr_ip: 10.2.2.2/32
+        tags:
+          - test: ec2_group_info_module
```

```suggestion
          - test: '{{ test_prefix }}ec2_group_info_module'
```

About 6 copies of the test will run in parallel, searching based on the tag will generate unpredictable results.

---

Ok. Fastest response ever ;)

---

Luck of the draw.

You marked your initial commit as a draft.  I'll wait for a comment from you that you're ready before I start looking at how thorough the tests are, but searching for static strings has caused me enough pain that I didn't want you to stare at failures wondering WTF is going on.

---

Yep, thanks for that! However `test_prefix` doesn't seem to be defined. Do I have to set this variable up myself somehow?

---

I guess it should be `resource_prefix` instead, will try that.